content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,378 @@
+
+{{< note >}}
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+will be removed in version 1.25.
+{{< /note >}}
+
+Pod Security admission `restricted` policy [can operate in several modes](/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces),
+`enforce`, `audit` or `warn` to help properly set the [security context](/docs/tasks/configure-pod-container/security-context/)
+according to security best practices. Nevertheless, pods' [security context](/docs/tasks/configure-pod-container/security-context/)
+should be separately investigated to limit the privileges and access pods may have.
+
+[Memory and CPU limits](/docs/concepts/configuration/manage-resources-containers/)
+should be set in order to restrict the memory and CPU resources a pod can
+consume on a node, and therefore prevent potential DoS attacks. It can be
+enforced by an admission controller.
+
+### Enabling Seccomp
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel
+syscall attack surface available inside containers. The seccomp filter mode
+leverages BPF to create an allow or deny list of specific syscalls, named
+profiles. Those seccomp profiles can be enabled on individual workloads,
+[a security tutorial is available](/docs/tutorials/security/seccomp/). In
+addition, the [Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project to facilitate the management and use of seccomp in clusters.
+
+Please note that Docker has been using
+[a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
+to only allow a restricted set of syscalls but Kubernetes is still not
+confining workloads by default. Fortunately,
+[Seccomp Default](/blog/2021/08/25/seccomp-default/), a new alpha feature to
+use a default seccomp profile for all workloads can now be enabled and tested.
+
+{{< note >}}
+Seccomp is only available on Linux nodes.
+{{< /note >}}
+
+### Enabling AppArmor or SELinux
+
+#### AppArmor
+
+[AppArmor](https://apparmor.net/) is a Linux kernel security module that can
+provide an easy way to implement Mandatory Access Control (MAC) and better
+auditing through system logs. To [enable AppArmor in Kubernetes](/docs/tutorials/security/apparmor/),
+at least version 1.4 is required. Like seccomp, AppArmor is also configured
+through profiles, where each profile is either running in enforcing mode, which
+blocks access to disallowed resources or complain mode, which only reports
+violations. AppArmor profiles are enforced on a per-container basis, with an
+annotation, allowing for processes to gain just the right privileges.
+
+{{< note >}}
+AppArmor is only available on Linux nodes, and enabled in
+[some Linux distributions](https://gitlab.com/apparmor/apparmor/-/wikis/home#distributions-and-ports).
+{{< /note >}}
+
+#### SELinux
+
+[SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) is also a
```

Comment:
```suggestion
[SELinux](http://selinuxproject.org/page/Main_Page) is also a
```

Reply:
I wanted to add that link at first but the page states "Note: This site is now somewhat historical: please visit https://github.com/SELinuxProject for up-to-date development information.". And the recommended link is a GitHub organization so I thought Wikipedia description would be better.

Reply:
Hmm I definitely missed the note on their page. I think it's okay to link to this: https://github.com/SELinuxProject/selinux or wiki

Reply:
Ok I guess it would Wikipedia or the SELinux notebook like this https://github.com/SELinuxProject/selinux-notebook/blob/main/src/selinux_overview.md for example. Because the user libraries and tools repo does not provide a lot of information about the general idea behind SELinux.

Reply:
```suggestion
[SELinux](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/selinux_overview.md) is also a
```

Reply:
Yes I love this update to SELinux notebook!! It also seems recently updated so less chance of it being stale